include/mpu_wrappers.h

```diff
@@ -177,7 +177,6 @@
     #define PRIVILEGED_FUNCTION
     #define PRIVILEGED_DATA
     #define FREERTOS_SYSTEM_CALL
-    #define portUSING_MPU_WRAPPERS    0

```

Comment:
why did we have to define it here to "0" and also in portable.h?

Reply:
Earlier the check was `#ifdef portUSING_MPU_WRAPPERS` and this line was used to define `portUSING_MPU_WRAPPERS` to `0` in case it was not defined. Now that the default definition is moved to `portable.h`, it is no longer needed and that is why, it is removed.